manual/program/proposal_guidelines.rst

```diff
@@ -0,0 +1,56 @@
+Proposal Submission Guidelines
+==============================
+
+.. image:: tiptap.gif
+  :width: 400
+  :alt: Meme cat writing a SciPy proposal
+
+Here are the instructions and tips for submitting a proposal to SciPy.
+
+Instructions
```

Comment:
These are great! I'm really happy to see these emphasized as I found them invaluable in the past when submitting proposals, and I think having a clear resource like this is excellent. :+1: